### ISSUE TITLE
It is possiable that mount the CESS client as a filesystem on Linux VFS 

### ISSUE THREAD

**Issue by g302ge** [CESSProject/cess-portal#12]
Hi, I am a new fans of decentrilized storage. These days, I am working for BeeGFS cache system. In my opinion, I belive that the decentrilized storage will be a filesystem which can be mount on current calculating stack for example Linux. Moreever, we can take advantage of Apache Arrow or other warehouse components to contribute a decentrilized warehouse. 

But the problem is that if CESS can be mount as a filesystem on VFS. I am willing to contribute this feature to CESS if it is possiable . 

Thx for ur reply soon later

**Comment by AstaFrode** [comment]
@g302ge  Yes, this is a great idea. At present, cess still relies on the Linux file system as its storage. If you can contribute this function to cess, we welcome you very much, and thank you very much for your contribution to the open source storage of cess. You can directly submit your pr.

**Comment by g302ge** [comment]
> @g302ge Yes, this is a great idea. At present, cess still relies on the Linux file system as its storage. If you can contribute this function to cess, we welcome you very much, and thank you very much for your contribution to the open source storage of cess. You can directly submit your pr.

Sure, I will work for this soon
